docs/reference/ImageGrab.rst

```diff
@@ -11,7 +11,7 @@ or the clipboard to a PIL image memory.
 
 .. versionadded:: 1.1.3
 
-.. py:function:: PIL.ImageGrab.grab(bbox=None)
+.. py:function:: PIL.ImageGrab.grab(bbox=None, includeLayeredWindows=False)
```

Comment:
Let's name this like `include_layered_windows` on the Python side.

Reply:
Okay, I've updated the commit